### ISSUE TITLE
I have a couple of questions about CCS

### ISSUE THREAD

**Issue by Dhyfer1** [SunsetTechuila/Cursor-Colors-Synchronizer#3]
Hi @SunsetTechuila 

First of all thanks for CCS, I like this repo and have it in my favorites. 😊
Now here are my questions:

1. After installing CCS is it normal for PowerShell to appear for more than 60 seconds after rebooting my PC? The window you see always appears after every reboot (did not appear before installing CCS), it appears after login and stays on the screen for more than 60 seconds, then closes on its own. The only thing it shows is a path in the title bar without any content. Isn't it better that this window opens and closes in a second, or better that it doesn't appear at all?
![Screenshot 2024-05-05 182118](https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/assets/50301201/acead40e-df7c-4014-be2d-eb7b5c768c59)
2. Are the cursors downloaded by CCS the free version or the paid version?

Thanks

**Comment by SunsetTechuila** [comment]
Hello!

> First of all thanks for CCS, I like this repo and have it in my favorites.

Nice to hear

> stays on the screen for more than 60 seconds, then closes on its own

This is not expected behavior. You have already tried uninstalling and then installing ccs again, haven't you?

> 2. Are the cursors downloaded by CCS the free version or the paid version?

https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/blob/ce51c9fcd0ccdaa4d1c9d7ef82cb7ea6e8f7beed/README.md?=1#L7

**Comment by Dhyfer1** [comment]
> This is not expected behavior. You have already tried uninstalling and then installing ccs again, haven't you?

Yes, I already tried it, that window still appears after reinstalling CCS (the number of seconds it remains on the screen after logging in always varies between 30 and 60 seconds), It even happened in previous versions as well. This window only disappears if I uninstall CCS

> 
> https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/blob/ce51c9fcd0ccdaa4d1c9d7ef82cb7ea6e8f7beed/README.md?=1#L7

Ok, now I understand. CCS uses the paid version of these cursors. I was confused because I saw in the readme that it said to use the paid version, but below you put the links to the free version instead of placing the links to the paid version, that's why I asked my question

**Comment by SunsetTechuila** [comment]
Could you run this cmd command as admin and tell what happens?
```cmd
powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
```

**Comment by Dhyfer1** [comment]
> Could you run this cmd command as admin and tell what happens?
> ```cmd
> powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
> ```

Of course. In Windows Terminal, with Command Prompt run as administrator, I get this:

```cmd
C:\Users\diego>powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows


Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
--     ----            -------------   -----         -----------     --------             -------
1      CursorThemeSync BackgroundJob   Running       True            localhost            ...
3      CursorColorSync BackgroundJob   Running       True            localhost            ...


PS C:\Users\diego>
```

While in PowerShell run as administrator, I get this:

```posh
PS C:\Users\diego> powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
The argument '%localappdata%\CCS\Listener.ps1' to the -File parameter does not exist. Provide the path to an existing '.ps1' file as an argument to the -File parameter.
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows

PS C:\Users\diego>
```

**Comment by Dhyfer1** [comment]
> Could you run this cmd command as admin and tell what happens?
> ```cmd
> powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
> ```

Of course. In Windows Terminal, with Command Prompt run as administrator, I get this:

```cmd
C:\Users\diego>powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows


Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
--     ----            -------------   -----         -----------     --------             -------
1      CursorThemeSync BackgroundJob   Running       True            localhost            ...
3      CursorColorSync BackgroundJob   Running       True            localhost            ...


PS C:\Users\diego>
```

While in PowerShell run as administrator, I get this:

```pwsh
PS C:\Users\diego> powershell.exe -ExecutionPolicy Bypass -NoExit -WindowStyle Hidden -File "%localappdata%\CCS\Listener.ps1"
The argument '%localappdata%\CCS\Listener.ps1' to the -File parameter does not exist. Provide the path to an existing '.ps1' file as an argument to the -File parameter.
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows

PS C:\Users\diego>
```

**Comment by SunsetTechuila** [comment]
nvm, try this in powershell
```powershell
iwr 'https://raw.githubusercontent.com/SunsetTechuila/Cursor-Colors-Synchronizer/dev/Installer.ps1' -useb | iex
```

**Comment by SunsetTechuila** [comment]
make sure you have ccs installed and then run this in powershell:

```powershell
iwr -useb "https://raw.githubusercontent.com/SunsetTechuila/Cursor-Colors-Synchronizer/dev/CCS.ps1" -out $env:LOCALAPPDATA\CCS\CCS.ps1
.$env:LOCALAPPDATA\CCS\CCS.ps1
```

tell me if this fixes the issue

**Comment by Dhyfer1** [comment]
> make sure you have ccs installed and then run this in powershell:
> 
> ```powershell
> iwr -useb "https://raw.githubusercontent.com/SunsetTechuila/Cursor-Colors-Synchronizer/dev/CCS.ps1" -out $env:LOCALAPPDATA\CCS\CCS.ps1
> .$env:LOCALAPPDATA\CCS\CCS.ps1
> ```
> 
> tell me if this fixes the issue

So I open PowerShell as administrator and ran this new command that you left me, after clicking Enter the installer prompts appears to select the type of cursor (tail or not tail) and the option to enable or disable background synchronization, and ends with the message: "Success! Enjoy."

Then I close PowerShell and restart the PC, but the issue is still not fixed. After logging in now this window appears for a few seconds (between 10 and 20) and then closes by itself:

![Screenshot 2024-05-06 174414](https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/assets/50301201/bdc67b61-ebc4-4db7-b8af-b91e0d307434)

After the previous window closes by itself, this other window appears but minimized in the taskbar, stays minimized for about 10 seconds and then closes by itself.

![Screenshot 2024-05-06 174427](https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/assets/50301201/fcd9af3b-037b-43e2-ba7a-198cb6e5b2ad)


Edit: After a few reboots, the time these two windows remain on screen always varies, but the CMD window always appears first followed by the minimized PowerShell window.

I also wanted to reinstall CCS to install it from scratch with the command you gave me, but the uninstaller it's not working with the new command. After running the uninstaller, this appears in PowerShell:

```powershell
PS C:\Users\diego> iwr 'https://raw.githubusercontent.com/SunsetTechuila/Cursor-Colors-Synchronizer/master/Uninstaller.ps1' -useb | iex
Reset-Cursor : Cannot validate argument on parameter 'Path'. The " Test-File -Path $PSItem " validation script for the
argument with value "%SystemRoot%\cursors\aero_working.ani" did not return a result of True. Determine why the
validation script failed, and then try the command again.
At line:24 char:1
+ Reset-Cursor
+ ~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Reset-Cursor], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Reset-Cursor

PS C:\Users\diego>
```

**Comment by SunsetTechuila** [comment]
Do you have any antivirus software other than windows defender?

**Comment by Dhyfer1** [comment]
No, only Windows Defender

**Comment by Dhyfer1** [comment]
No, only Windows Defender

**Comment by SunsetTechuila** [comment]
I've published a new release, which should fix your issue https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/releases/tag/v3.2

**Comment by Dhyfer1** [comment]
> I've published a new release, which should fix your issue https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/releases/tag/v3.2

Ok. How do I install it? Do I use the installation command in the Readme or do I use the [other command](https://github.com/SunsetTechuila/Cursor-Colors-Synchronizer/issues/3#issuecomment-2096696732) you gave me?

Edit: I have used the command in the readme file and no more CMD or PowerShell window appears after reboot, problem solved. Thank you very much😊

**Comment by SunsetTechuila** [comment]
fixed by 7099065

**Comment by SunsetTechuila** [comment]
> After running the uninstaller, this appears in PowerShell:

fixed by e65c080
